configure.ac

```diff
@@ -207,8 +207,19 @@ CFLAGS="$CFLAGS $SDL_CFLAGS"
 LIBS="$LIBS $SDL_LIBS"
 
 if (test x$enable_jpg = xyes || test x$enable_tif = xyes) && test x$enable_imageio != xyes; then
-    AC_CHECK_HEADER([jpeglib.h], [have_jpg_hdr=yes])
-    AC_CHECK_LIB([jpeg], [jpeg_CreateDecompress], [have_jpg_lib=yes])
+    PKG_CHECK_MODULES([LIBJPEG], [libjpeg], [dnl
```

Comment:
This doesn't work for systems that have the libraries installed, but don't have pkgconfig.

Reply:
It should fall back on the old detection method in that case, similarly to what the libpng and libwebp checks do.

Reply:
Oh, right, thanks. :)